chapters/functions.tex

```diff
@@ -542,7 +542,9 @@ \subsection{Functional Input Arguments to Functions}\label{functional-input-argu
   input Integrand integrand; // Integrand is a partial function type
   output Real integral;
 algorithm
-  integral := quadrature(x1, (x1+x2)/2, integrand)+  quadrature((x1+x2)/2, x2, integrand);
+  integral :=
+    quadrature(x1, (x1 + x2) / 2, integrand) +
+      quadrature((x1 + x2) / 2, x2, integrand);
```

Comment:
It's because it's a continuation of the line above.

Reply:
To me both line 546 and 547 are continuations of line 545 and should be intended the same.
The '+'sign does not really make line 547 subordinate to line 546.

Reply:
OK, then we simply have different ideas about how to indent when splitting an expression over several lines.  My inclination to indent comes from situations like this:
```
integral :=
  foo(
    quadrature(x1, (x1 + x2) / 2, integrand) +
      quadrature((x1 + x2) / 2, x2, integrand),
    quadrature(x1, (x1 + x2) / 2, integrand),
    quadrature((x1 + x2) / 2, x2, integrand)
  );
```
If you insist I'll do it your way just so that we can move on.

Reply:
Yes, it seems we have different ideas.
In the second example I could understand that indentation, but would have preferred (also) using named arguments or other alternatives - as the indentation itself doesn't suffice to make it easy to read.